packages/frontend-2/components/projects/Dashboard.vue

```diff
@@ -118,7 +118,7 @@ const filterProjectsToMove = ref(false)
 const openNewProject = ref(false)
 const showLoadingBar = ref(false)
 const showMoveProjectDialog = ref(false)
-const emittedProjectId = ref('')
+const emittedProjectId = ref<Nullable<string>>(null)
```

Comment:
Could this not be `ref<string | undefined>(undefined)`  in order to avoid the piping for the prop?